gradle/libs.versions.toml

```diff
@@ -17,7 +17,7 @@ junit5 = "5.10.2"
 # https://kotlinlang.org/docs/releases.html#release-details
 kotlin = "1.9.23"
 # set in <root>/settings.gradle.kts
-kotlinCoroutines = "1.7.3"
+kotlinCoroutines = "1.8.0"
```

Comment:
coroutines can't be safely bumped like this
https://github.com/aws/aws-toolkit-jetbrains/blob/main/kotlinResolution.settings.gradle.kts

Reply:
Sure